TriblerGUI/widgets/trustpage/js/drawing.js

```diff
@@ -28,48 +28,102 @@ function drawNodes(svg, data, on_click) {
     // Always remove existing nodes before adding new ones
     selectNodes(svg).remove();
 
+    // Do not display the label after the graph is updated
+    hoverInfoLabel.style("display", "none");
```

Comment:
This does not belong here, please move to main.js right after the call to drawNodes()